.github/workflows/docs.yaml

```diff
@@ -25,21 +25,28 @@ jobs:
     - name: Checkout Code
       uses: actions/checkout@v4
 
-    - name: Set up Python 3.10
```

Comment:
Since we are using a `uv venv`, we don't need the Python toolchain setup in the GH actions.

Reply:
oh nice; need to make sure the venv is created with python 3.10

Reply:
Hmm, yeah, maybe I should add it back. it only can create a venv from a Python interpreter on the system path. So I should ensure its there on the GH actions runner.

Think I got lucky in that it worked while removing it, their base env must just have 3.10